src/components/Footer.tsx

```diff
@@ -0,0 +1,142 @@
+import EmailIcon from '@mui/icons-material/Email';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import { Box, Typography, Link, Icon, useMediaQuery } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+
+import { GradientBorderDivider } from '@components';
+import { FooterResponse, Network } from '@utils/types';
+import theme from 'theme';
+
+import SlackIcon from '../../public/icons/slack-icon.svg';
+import { addLineBreakAfterPeriod } from '@utils/helpers';
+
+const iconNetworkMapper = {
+  linkedIn: <LinkedInIcon fontSize="large" />,
+  github: <GitHubIcon fontSize="large" />,
+  instagram: <InstagramIcon fontSize="large" />,
+  twitter: <TwitterIcon fontSize="large" />,
+  email: <EmailIcon fontSize="large" />,
+  slack: <SlackIcon width={32} height={32} fill={'FFF'} />,
+};
+
+export const Footer = ({
+  title,
+  subtitle,
+  description,
+  network,
+  link,
+}: FooterResponse) => {
+  const isMobile = useMediaQuery(theme.breakpoints.down('md'));
+  const renderSocialNetworkIcon = (network: Network) => {
+    return (
+      <a href={network.link}>
+        <Icon
+          sx={{
+            color: 'primary.dark',
+            marginRight: '7px',
+            width: '34px',
+            height: '34px',
+          }}
+          fontSize="large"
+        >
+          {iconNetworkMapper[network.type as keyof typeof iconNetworkMapper]}
+        </Icon>
+      </a>
+    );
+  };
+
+  const modifyFooterDescription: string = addLineBreakAfterPeriod(description);
+  return (
+    <>
+      <GradientBorderDivider
+        height="1rem"
+        width="100%"
+        gradientColors="linear-gradient(to right, #84B1D0, #FFDEA6, #FFB59D)"
+      />
+      <Box
+        sx={{
+          bgcolor: '#F4F0EF',
+          py: { xs: 2, md: 5 },
+          px: { xs: 2, md: 10, xl: '20%' },
+          display: 'flex',
+          flexDirection: { xs: 'column', md: 'row' },
+          alignItems: 'flex-start',
+          justifyContent: 'space-between',
+          pb: { xs: 5 },
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'flex-start',
+            mb: { xs: 2, md: 0 },
+          }}
+        >
+          <Image
+            src="/logo_white.png"
+            alt="Woman Coding Community"
+            width={80}
+            height={80}
+          />
+
+          <p
+            style={{ fontSize: isMobile ? '12px' : '14px' }}
+            dangerouslySetInnerHTML={{ __html: modifyFooterDescription }}
+          />
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'flex-start',
+          }}
+        >
+          <Typography
+            sx={{ fontSize: '16px' }}
```

Comment:
Could we reuse the fontsize as variable and re-use? the same goes to colours and fontWeight, that would make easier to change later in case we want all pages to have the same size and colour.